src/components/BSelect/BSelect.md

```diff
@@ -0,0 +1,34 @@
+```js
+
+<div>
+  <div style="width: 50%">
+    <b-select
+      :required=false
```

Comment:
Can required be set by just having "required" as a property without a value?

Reply:
I'll add this in my refactor branch